reconcile/utils/ocm.py

```diff
@@ -931,11 +931,36 @@ def _init_blocked_versions(self, blocked_versions):
                 raise TypeError(f"blocked version is not a valid regex expression: {b}")
 
     @retry(max_attempts=10)
-    def _get_json(self, api):
-        r = requests.get(f"{self.url}{api}", headers=self.headers)
+    def _do_get_request(self, api: str, params: dict[str, str]):
+        r = requests.get(f"{self.url}{api}", headers=self.headers, params=params)
         r.raise_for_status()
         return r.json()
 
+    def _get_json(self, api):
+        def response_is_list(rs: dict[str, Any]) -> bool:
+            return rs["kind"].endswith("List")
```

Comment:
Move this itnernal method to the class (make it a `@staticmethod` if you need to). It'll make testing easier.

Reply:
Good point. done